docs/docs/installation.md

```diff
@@ -81,13 +81,17 @@ and compile it.
  git clone https://bitbucket.org/huhabla/temporal-raster-sampling.git t.rast.sample
  cd t.rast.sample
  make
+ ``` 
+> **_HINT:_** run : `sudo apt-get install python-is-python3` If you get the error: `/usr/bin/env: ‘python’: No such file or directory
+` in the logs from `make` command.
```

Comment:
Please also add to hint that this will change the python version system wide and python2 will not be used anymore. Therefore it is a bit dangerous. How can this be reverted if python2 is needed for old projects? If there is no solution to switch python version with this package (I don't know it), please use https://linuxhint.com/update_alternatives_ubuntu/ instead. It allows to simply configure to which version python should point to.